CONTRIBUTING.md

```diff
@@ -101,7 +101,7 @@ Then add a title and description to your pull request that explains your contrib
     
     #### How to do that? 
 
-    - You can do that by hosting all you images and screenshots to any images hosting site such as [imgur]('https://imgur.com/'), [imgbb]('https://imgbb.com/'), [postimages]('https://postimages.org/').
+    - You can do that by hosting all you images and screenshots to any images hosting sites such as [imgur]('https://imgur.com/'), [imgbb]('https://imgbb.com/'), [postimages]('https://postimages.org/').

```

Comment:
I asked you to get rid of quotes, not put quotes everywhere

```
[imgur](https://imgur.com/)
[imgbb](https://imgbb.com/)
```

This is what you should do.

Please get this done

Reply:
@pi-squared-4 Just this small change